drivers/counter/counter_mcux_rtc.c

```diff
@@ -236,6 +236,14 @@ static int mcux_rtc_init(const struct device *dev)
 	RTC_GetDefaultConfig(&rtc_config);
 	RTC_Init(config->base, &rtc_config);
 
+#ifdef CONFIG_SEL_LPO_1KHZ_CLOCK
+	/* The RTC prescaler increments using the LPO 1 kHz clock intead of RTC 32.768 kHz clock.
+	 * frdm_ke17z512 has no 32KHZ OSC, the RTC clock comes from SOSC, RTC_CLKIN, LPO 1KHZ.
+	 * But usually the SOSC is connected with 8MHZ oscilator, so only 1kHZ LPO is useable.
+	 */
+	config->base->CR |= RTC_CR_LPOS_MASK;
```

Comment:
use `RTC_EnableLPOClock()` instead, already available in `fsl_rtc.h`

Reply:
Done